packages/fxa-admin-server/src/graphql.ts

```diff
@@ -120,7 +120,7 @@ export interface MozSubscription {
     cancelAtPeriodEnd: boolean;
     endedAt?: Nullable<number>;
     latestInvoice: string;
-    manageSubscriptionLink: string;
+    manageSubscriptionLink?: Nullable<string>;
```

Comment:
This is the code gen'd file and as result this change wouldn't really presist (see note at top of file). You want to change in the actual model which I believe is [here](https://github.com/mozilla/fxa/blob/c3a23cec5ac42af4dfc1c8852c224c3f4a75cd32/packages/fxa-admin-server/src/gql/model/moz-subscription.model.ts#L29).

Reply:
haha oops thanks.  That'll teach me not to grep and start editing the most obvious looking thing.